source/globalCommands.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: UTF-8 -*-
+﻿# -*- coding: UTF-8 -*-

```

Comment:
Somehow, a weird character ended up at the start of this line. Could you have saved the file as UTF-8 with BOM? You might as well add your name to the copyright line in the process.